components/match2/wikis/valorant/match_group_input_custom.lua

```diff
@@ -242,12 +242,31 @@ function matchFunctions.mergeWithStandalone(match)
 		return match
 	end
 
+	-- Update Opponents from the Stanlone Match
 	match.opponent1 = standaloneMatch.match2opponents[1]
 	match.opponent2 = standaloneMatch.match2opponents[2]
 
-	local match2games = standaloneMatch.match2games
-	for i = 1, #match2games do
-		match['map' .. i] = match2games[i]
+	-- Update Maps from the Standalone Match
+	for index, game in ipairs(standaloneMatch.match2games) do
+		-- JSON encoded fields has to be parsed first
+		game.participants = Json.parseIfString(game.participants)
+		game.extradata = Json.parseIfString(game.extradata)
+		-- Save the map into current match object
+		match['map' .. index] = game
```

Comment:
```suggestion
	-- Update Maps from the Standalone Match
	for index, game in ipairs(standaloneMatch.match2games) do
		game.participants = Json.parseIfString(game.participants)
		game.extradata = Json.parseIfString(game.extradata)
		match['map' .. index] = game
```

No need for these comments.

For the top comment you could probably argue it should be made into a function instead `updateMapsFromStandaloneMatch` but let's keep it this way.